README.md

```diff
@@ -69,26 +69,21 @@ On ArchLinux:
 sudo pacman -S  base-devel rsync opam pandoc jre8-openjdk mpfr maven
 ```
 
-On OSX, using [Homebrew](https://brew.sh/):
+On OSX, using [Homebrew](https://brew.sh/), after installing XCode:
 
 ```sh
 brew tap caskroom/cask caskroom/version
 brew cask install java8
-brew install bison flex libtool autoconf gmp mpfr pandoc maven opam
-```
-
-Note that Homebrew install flex as a keg-only package. To install flex's library, use:
-
-```sh
-brew link --force flex
+brew install automake libtool gmp mpfr pkg-config pandoc maven opam python z3
```

Comment:
OSX comes with python 2.7 pre-installed. Not sure you'd need to install it via homebrew.

Reply:
I don't really know what to say except that python wasn't in the path when I built evm on a freshly upgraded high Sierra today, and this fixed it. Are you sure that's a correct statement?